src/components/Dashboard/ViewDashboard.js

```diff
@@ -27,24 +28,22 @@ export const ViewDashboard = props => {
     }, [props.dashboardIsEditing, props.dashboardIsPrinting])
 
     useEffect(() => {
-        document.querySelector('.dashboard-wrapper')?.scroll(0, 0)
+        Array.from(
+            document.getElementsByClassName('dashboard-scroll-container')
+        ).forEach(container => {
+            container.scroll(0, 0)
+        })
```

Comment:
Why not using `querySelector` still?
I understand there is always only 1 element with that class.
Or maybe use this instead?
```suggestion
            document.getElementsByClassName('dashboard-scroll-container').item(0)?.scroll(0,0)
```

Reply:
Actually, there can be 2 elements with the class

Reply:
Yes, i missed the one in `DashboardContainer`.
Nvm the comments.